_docs_integrate/error-codes.md

```diff
@@ -24,75 +24,144 @@ required_by:
-| {% include anchor a="error.transport.notSupported" %}                                                       | The requested method is not yet supported.                                                                                                                                                                                                                                |
-| {% include anchor a="error.transport.recordNotFound" %}                                                     | The requested record was not found.                                                                                                                                                                                                                                       |
-| {% include anchor a="error.transport.relationships.wrongChangeStatus" %}                                    | The Relationship change has the wrong status to run the requested operation.                                                                                                                                                                                              |
-| {% include anchor a="error.transport.secrets.secretNotFound" %}                                             | No secret was found for a specific type.                                                                                                                                                                                                                                  |
-| {% include anchor a="error.transport.secrets.wrongBaseKeyType" %}                                           | The given secret type is not supported!                                                                                                                                                                                                                                   |
-| {% include anchor a="error.transport.signatureNotValid" %}                                                  | A signature is invalid.                                                                                                                                                                                                                                                   |
-| {% include anchor a="error.transport.tokens.invalidTokenContent" %}                                         | The given token content is invalid.                                                                                                                                                                                                                                       |
+# Table of error codes
```

Comment:
why do we need these lines? I think you forgot to delete them from the version with the multiple tables.. We don't want to talk about "libraries" here.

(this was just merged.. please do another PR)

Reply:
I added it only because otherwise the table of content 'On this page' is empty and that doesn't look good in my opinion.

Reply:
then you can disable toc :)

Reply:
I haven't done it yet because I had something else to finish for the demo-team but I will do it after my vacation.

Reply:
all good, fixed in #248